src/Unit-3/README.md

```diff
@@ -45,7 +45,8 @@ If you have any questions about Octokit or want to learn more about it, make sur
 ## Things you'll need
 **You will need to make a GitHub OAuth access token for the API**.
 
-This will be a throwaway token in your account only used for this app. [Follow the instructions here](https://help.github.com/articles/creating-an-access-token-for-command-line-use/) and write down your OAuth token.
+This will be a throwaway token in your account only used for this app. [Follow the instructions here](https://help.github.com/articles/creating-an-access-token-for-command-line-use/) and write down your OAuth token. When you create your access token, you can select repo as your scope. This scope will be all you need for this project.  
+![](../../images/OAuth_Scope.png)
```

Comment:
You need to start adding comments to your markdown images @izavala - need it for folks who are using screen-readers or can't otherwise load images.

Reply:
Definitely makes sense,  I will make sure to add a description to every image going forward